test/integration_test.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from requests.models import Response
+from app.create_app import create_app
+from dotenv import find_dotenv, load_dotenv
+from unittest.mock import Mock, patch
+
+@pytest.fixture
+def client():
+    # Use our test integration config instead of the 'real' version 
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+
+    #Create the new app.
+    test_app = create_app()
+    
+    # Use the app to create a test_client that can be used in our tests.
+    with test_app.test_client() as client: 
+        yield client
+
+@patch('requests.get')
+def test_index_page(mock_get_requests, client):
+    mock_item_response = Response()
+    mock_item_response.status_code = 200
+    mock_item_response._content = b'[{"id": "123", "idList": "987", "name": "Why hello there", "dateLastActivity": "2021-01-06T21:14:06.518Z"}]'
+
+    mock_list_response = Response()
+    mock_list_response.status_code = 200
+    mock_list_response._content = b'[{"id": "987", "name": "Done"}]'
+
+    mock_get_requests.side_effect = [mock_item_response, mock_list_response]
```

Comment:
It would be more robust to make side_effect a function that checks the requested URL and then decides what to return
- Your test would validate that the app makes requests to the correct URLs
- Your test would not be dependent on the order that the app makes requests in.